db/migrate/20211118110017_add_description_to_outifts.rb

```diff
@@ -0,0 +1,5 @@
+class AddDescriptionToOutifts < ActiveRecord::Migration[6.0]
+  def change
+    add_column :outfits, :description, :string
```

Comment:
@emilyunderhill shall we make the data type :text instead of :string? so more characters can be typed and also i think this renders the box input in the form to be larger...